common/item_instance.h

```diff
@@ -301,6 +301,7 @@ namespace EQ
 		int GetItemHaste(bool augments = false) const;
 		uint32 GetItemGuildFavor() const;
 		std::vector<uint32> GetAugmentIDs() const;
+		std::string DetermineMoneyStringForParcels(uint32 cp) const;
```

Comment:
Done

Reply:
Ah, I guess the server side string function would still be needed for any messages that are supposed to show the amount. It was a bad suggestion to remove it completely if it's still needed for those. Otherwise looks fine even if that function needs to be re-added.